packages/expo/src/__tests__/Haptic-test.ts

```diff
@@ -0,0 +1,89 @@
+import { UnavailabilityError } from 'expo-errors';
+import { NativeModules, Platform } from 'react-native';
+
+import {
+  mockPlatformAndroid,
+  mockPlatformWeb,
+  mockProperty,
+  unmockAllProperties,
+} from '../../test/mocking';
+import * as Haptic from '../Haptic/Haptic';
+
+async function executeUnavailableMethod(method, name) {
```

Comment:
This function is used only once in the entire file, maybe inlining it would improve readability? 🙂 (Looking at the file in the code editor prolly worked ok, but with some comments in GitHub PR page declaration and its use float apart 😅)